server/start.ts

```diff
@@ -82,6 +86,7 @@ async function main() {
     transactions(pool, provider),
     balances(pool, provider),
     transfers(NOUNS_TOKEN_ADDRESS, pool, provider),
+    socials(pool),
```

Comment:
we need to gracefully handle when `AIRSTACK_API_KEY` is not available. Let's not start this indexer if `AIRSTACK_API_KEY` is not set